### PR TITLE
Add `librewolf-app`

### PR DESCRIPTION
**Pull request by wizard-28**
Adds librewolf's appimage

### CODE REVIEW

**Comment by wizard-28** [comment]
I couldn't test in locally (storage space issues). So test it yourselves before merging.

File: packages/librewolf-app/librewolf-app.pacscript [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+removescript() {
+  # Remove .desktop file
+  rm -f ~/.local/share/applications/userapp-LibreWolf-R8NL90.desktop
+
+  # Delete .librewolf
+  if ask "Do you want to delete ~/.librewolf/ files?" Y; then
+    rm -rf ~/.librewolf/
+  fi
```
Comment (Henryws) [inline packages/librewolf-app/librewolf-app.pacscript:39]: This is not allowed in pacstall

Comment (wizard-28) [reply packages/librewolf-app/librewolf-app.pacscript:39]: Well... You merged `walc-app`, then you didn't have any issues?

**Comment by Henryws** [comment]
> I couldn't test in locally (storage space issues). So test it yourselves before merging.

It works, but could you add a `.desktop` file to it so it's easier to launch?

**Comment by wizard-28** [comment]
> > I couldn't test in locally (storage space issues). So test it yourselves before merging.
> 
> It works, but could you add a `.desktop` file to it so it's easier to launch?

It's auto generated on first launch. See I'm even removing it during uninstall.
https://github.com/pacstall/pacstall-programs/blob/16871cc351e67212e1c615e91e9cb94a7b1192d9/packages/librewolf-app/librewolf-app.pacscript#L34

Can you check with KDE or GNOME if it appears in the app launcher?

**Comment by Henryws** [comment]
> It's auto generated on first launch. See I'm even removing it during uninstall.

Doesn't seem to auto generate with me and rofi

**Comment by wizard-28** [comment]
> > I couldn't test in locally (storage space issues). So test it yourselves before merging.
> 
> It works, but could you add a `.desktop` file to it so it's easier to launch?

Test the uninstallation too.

**Comment by Henryws** [comment]
p

> Test the uninstallation too.

It works

**Comment by wizard-28** [comment]
> > It's auto generated on first launch. See I'm even removing it during uninstall.
> 
> Doesn't seem to auto generate with me and rofi

Well... it worked for me...
```py
~ via 🐍 v3.8.10
❯ cat .local/share/applications/userapp-LibreWolf-R8NL90.desktop
[Desktop Entry]
Encoding=UTF-8
Version=1.0
Type=Application
NoDisplay=true
Exec=/tmp/.mount_librewkml74t/librewolf %u
Name=LibreWolf
Comment=Custom definition for LibreWolf
```

**Comment by wizard-28** [comment]
> > It's auto generated on first launch. See I'm even removing it during uninstall.
> 
> Doesn't seem to auto generate with me and rofi

Did you launch it manually for the first time? It'll auto generate then.

**Comment by Henryws** [comment]
> > > It's auto generated on first launch. See I'm even removing it during uninstall.
> > 
> > 
> > Doesn't seem to auto generate with me and rofi
> 
> Did you launch it manually for the first time? It'll auto generate then.

I launched `/usr/bin/librewolf` which appeared to be a symlink

**Comment by wizard-28** [comment]
> > > > It's auto generated on first launch. See I'm even removing it during uninstall.
> > > 
> > > 
> > > Doesn't seem to auto generate with me and rofi
> > 
> > 
> > Did you launch it manually for the first time? It'll auto generate then.
> 
> I launched `/usr/bin/librewolf` which appeared to be a symlink

Well I launched GNOME, it doesn't appear to be in the menu system. So do you want me to manually install a `.desktop` file? If I do so, then I would also have to download the `.png` file of the LibreWolf logo. Otherwise the launcher won't have an icon!

**Comment by Henryws** [comment]
> So do you want me to manually install a `.desktop` file? If I do so, then I would also have to download the `.png` file of the LibreWolf logo. Otherwise the launcher won't have an icon!

Yes please!

**Comment by wizard-28** [comment]
> > So do you want me to manually install a `.desktop` file? If I do so, then I would also have to download the `.png` file of the LibreWolf logo. Otherwise the launcher won't have an icon!
> 
> Yes please!

Can you link me the logo?

**Comment by Henryws** [comment]
> Can you link me the logo?

https://librewolf-community.gitlab.io/images/logo.png

**Review by Henryws — APPROVED** [review]

**Pull request merged**